lms/djangoapps/teams/api.py

```diff
@@ -271,10 +272,35 @@ def get_team_for_user_course_topic(user, course_id, topic_id):
         logger.error(msg.format(
             username=user.username,
             course=course_id,
-            topic=topic_id,
+            topic=teamset_id,
         ))
         return CourseTeam.objects.filter(
             course_id=course_key,
             membership__user__username=user.username,
-            topic_id=topic_id,
+            topic_id=teamset_id,
         ).first()
+
+
+def get_teamset(course_id, teamset_id):
+    """
+    Returns the TeamsetConfig associated with this course_id and teamset_id
```

Comment:
slight nitpick: the method is called get_teamset, while the header comments uses "TeamsetConfig". Perhaps pick one?